documentation/code-generation.md

```diff
@@ -94,3 +94,70 @@ generator –r –v –l=temp –version=v11.2.0-beta C:/azure-rest-api-specs/sp
 ## Generate Azure SDK for Go profiles
 
 Take a look into the [profile generator documentation](https://github.com/Azure/azure-sdk-for-go/tree/main/eng/tools/profileBuilder)
+
+## Generate Go Mgmt SDK from Typespec
+### Prerequisites
+- [Node.js 18.x LTS](https://nodejs.org/en/download) or later
+- [Go 1.23.x](https://go.dev/doc/install) or later
+- [Git](https://git-scm.com/downloads)
+- Identify the `tspconfig.yaml` file for your package in the [Rest API Spec Repo](https://github.com/Azure/azure-rest-api-specs) and ensure there is a configuration for the Go SDK similar to that shown below and in [this example](https://github.com/Azure/azure-rest-api-specs/blob/b09c9ec927456021dc549e111fa2cac3b4b00659/specification/contosowidgetmanager/Contoso.Management/tspconfig.yaml#L40)
```

Comment:
do we need to lock to the specific commit it? i updated the config template last week. shall we keep it use `main`?

Reply:
@chidozieononiwu let's use the `main` in the link to refer to the latest version.